--- Makefile.in~	Tue Mar 28 07:18:39 1995
+++ Makefile.in	Wed Jun  2 21:49:20 1999
@@ -57,8 +57,8 @@
 	cd filters; make
 
 install: magicfilter magicfilter.man
-	${INSTALL_EXEC} magicfilter ${bindir}/magicfilter
-	${INSTALL_DATA} magicfilter.man ${mandir}/magicfilter.${mansec}
+	${INSTALL_EXEC} magicfilter $(DESTDIR)${bindir}/magicfilter
+	${INSTALL_DATA} magicfilter.man $(DESTDIR)${mandir}/magicfilter.${mansec}
 
 install_filters:
 	cd filters; make install
--- filters/Makefile.in~	Wed Jun  2 21:46:59 1999
+++ filters/Makefile.in	Wed Jun  2 21:50:51 1999
@@ -69,7 +69,7 @@
 
 install: all
 	for filter in ${FILTERS}; do \
-		${INSTALL_EXEC} $$filter ${bindir}/$$filter; \
+		${INSTALL_EXEC} $$filter $(DESTDIR)${bindir}/$$filter; \
 	done
 
 clean:
